python/paddle/fluid/tests/unittests/dygraph_to_static/ifelse_simple_func.py

```diff
@@ -28,6 +28,28 @@ def loss_fn(x, lable):
     return loss
 
 
+def dyfunc_ifExp_with_while(x):
+    y = [x]
+
+    def cond(i, ten, y):
+        return i < ten
+
+    def map_func(func, tensor_list):
+        return [func(x) for x in tensor_list]
+
+    def body(i, ten, y):
+        # It will be converted into `layers.cond` as followed.
+        # map_func(lambda x: fluid.layers.cond(i==0, lambda: x, lambda: add_fn(x), y)
+        y = map_func(lambda x: x if i == 0 else add_fn(x), y)
+        i += 1
+        return [i, ten, y]
+
+    i = fluid.layers.fill_constant(shape=[1], dtype='int64', value=0)
+    ten = fluid.layers.fill_constant(shape=[1], dtype='int64', value=10)
+    i, ten, y = fluid.layers.while_loop(cond, body, [i, ten, y])
```

Comment:
It is dyfunc but with static `while_loop`, is that due to your test only run `IfElseTransformer` so you won't like to use python `while`? If so I suggest you to move this function to the unit test where you use the function and comment about it.

In my understanding, you organized `ifelse_simple_func.py` as dyfunc which can be transformed, not "partial transformed" function like this func

Reply:
Thanks, fixed.